README.md

```diff
@@ -52,13 +52,19 @@ This is a work-in-progress implementation of a service broker for services provi
 `PASSWORD` - username password
 `DB_PREFIX` - a prefix that can be used to tag instances. Defaults to `compose-broker`
 `CLUSTER_NAME` - a name of your enterprise cluster if you've got one and want to use it. Defaults to hosted compose
-`ACCESS_TOKEN` - your API key for Compose.
+`COMPOSE_ACCESS_TOKEN` - your API key for Compose.
 
-## Running tests locally
+## Running local tests
 
-The integration tests communicate with MongoDB server(s). You will need to run a MongoDB server on `localhost` before running local (non-Compose) tests.
+You **must** have at least version 1.4.0 of ginkgo installed
 
 ```
-docker run -p 27017:27017 --name mongo -d mongo:3.2
 ./scripts/run-local-tests.sh
 ```
+## Running all tests, including using live Compose API
+
+You **must** have at least version 1.4.0 of ginkgo installed
```

Comment:
Agreed. I'll tidy this up.

I'm not really a massive fan of the two scripts that do the same thing, however I believe #7 is currently expecting the scripts to exist, so thought it best to leave as is for now.